Automattic/studio-demo-site-companion
# Add notice on the frontend

- Related: https://github.com/Automattic/dotcom-forge/issues/5513

## Description

Add notice in the front page of demo sites.
The notice can be closed and will write a cookie to avoid showing it next times for that user/site.
Clicking on Try  will open a new tab in the browser.

## Testing instructions

* Add this plugin file `studio-demo-site-companion.php` directly into the `wp-content/mu-plugins/` folder in any website. For example in Studio, wp-now or production. You'll have only one file in `wp-content/mu-plugins/studio-demo-site-companion.php`
* Navigate to `/`
* Observe the changes mentioned in the description
* Access `/wp-admin`
* Observe the notice and the new background color.

<img width="1268" alt="Screenshot 2024-04-22 at 15 54 16" src="https://github.com/Automattic/studio-demo-site-companion/assets/779993/39fd3f5c-b809-44cf-8a2f-900c808aa222">

<img width="1267" alt="Screenshot 2024-04-22 at 15 54 24" src="https://github.com/Automattic/studio-demo-site-companion/assets/779993/287df30d-f651-4e08-9176-369c6dda0c6c">
